learn/wme-backup-restore/backup/externalinstance/backup-aws.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Backup on AWS"
+id: ""
+sidebar_label: "Backup on AWS"
+---
+---
+
+
+Stop WME  EC2 external instance and detach network interface [follow the steps given here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#detach_eni), make a note of interface ID or eni ID.
```

Comment:
In any Provider we can take backups from directories as well ?
so you can mention it and write the same like ESxi